Formula/kmc.rb

```diff
@@ -0,0 +1,32 @@
+class Kmc < Formula
+  # cite Deorowicz_2015: "https://doi.org/10.1093/bioinformatics/btv022"
+  desc "Fast and frugal disk based k-mer counter"
+  homepage "http://sun.aei.polsl.pl/kmc/"
+  url "https://github.com/marekkokot/KMC/archive/v3.0.1.tar.gz"
+  sha256 "0dbc9254f95541a060d94076d2aa03bb57eb2da114895848f65af0db1e4f8b67"
+  head "https://github.com/marekkokot/KMC.git"
+
+  depends_on "gcc" # needs OpenMP
```

Comment:
```ruby
depends_on "gcc" if OS.mac? # needs OpenMP
```

Reply:
Ok. A `needs :openmp` would be handy here :-P